init.lua

```diff
@@ -91,9 +115,10 @@ minetest.register_node("trash_can:trash_can_wooden",{
 			"size[8,9]" ..
 			"button[0,0;2,1;empty;Empty Trash]" ..
 			"list[context;trashlist;3,1;2,3;]" ..
-			"list[current_player;main;0,5;8,4;]"
+			"list[current_player;main;0,5;8,4;]" ..
+			moditems.BOXART
 		)
-		meta:set_string("infotext", "Trash Can")
+		meta:set_string("infotext", moditems.INFOBOX_CAN)
```

Comment:
`INFOBOX_CAN` is a table in the MCL2 version. `set_string` does not work (properly?) with tables.